test/test_stanfit.py

```diff
@@ -69,15 +68,23 @@ def test_validate_good_run(self):
 
         df = fit.get_drawset()
         self.assertEqual(
-            df.shape,
-            (
-                fit.chains * fit.draws,
-                len(fit.column_names),
-            ),
+            df.shape, (fit.chains * fit.draws, len(fit.column_names))
         )
         _ = fit.summary()
 
-        self.assertIsNone(fit.diagnose())
+        # TODO - use cmdstan test files instead
```

Comment:
thanks for updating unit tests too - CmdStan release 2.20 has better diagnostics on `bin/diagnose`